easybuild/easyconfigs/d/double-conversion/double-conversion-3.0.3-foss-2018a.eb

```diff
@@ -13,15 +13,24 @@ sources = ['v%(version)s.tar.gz']
 checksums = ['ac84e00c19fdb5ee20c01feb18b0fa2e938bf4b8cf71d43bab62502c8b65420a']
 
 builddependencies = [
-    ('SCons', '3.0.1', '-Python-3.6.4'),
+    ('CMake', '3.12.1', '', ('GCCcore', '6.4.0')),
 ]
 
-installopts = "DESTDIR=%(installdir)s prefix='' && "
-installopts += "mkdir %(installdir)s/include && cp double-conversion/*.h %(installdir)s/include"
+separate_build_dir = True
+
+local_buildtype = '-DCMAKE_BUILD_TYPE=Release '
+configopts = [
+    local_buildtype + c for c in (
+        '',
+        '-DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_STATIC_LIBRARY_SUFFIX_CXX=_pic.a',
+        '-DBUILD_SHARED_LIBS=ON'
+    )
+]
 
 sanity_check_paths = {
-    'files': ['include/double-conversion.h', 'include/utils.h', 'lib/libdouble-conversion.a',
-              'lib/libdouble-conversion.%s' % SHLIB_EXT, 'lib/libdouble-conversion_pic.a'],
+    'files': ['include/double-conversion/double-conversion.h', 'include/double-conversion/utils.h',
```

Comment:
@Flamefire How come the header files are now in a `double-conversion` subdirectory?

Reply:
Because this is the correct directory. Previously scons didn't even install the headers and they were copied manually to a directory which I see no reasoning of, see https://github.com/easybuilders/easybuild-easyconfigs/pull/9659/files#diff-113da115196d50f313926fb7149ba44dL20

This is the directory CMake installs the headers to and this library itself used to include the headers by `double-conversion/double-conversion.h` format, although this has been changed, see https://github.com/google/double-conversion/commit/04c6f0a70336b4c4a71e25a48f9ee58ec0a9321d. Hence I'm sure that this is now the correct directory layout.

Reply:
Hmm, if this was wrong, I wonder why this didn't break the `Qt5` easyconfigs where `double-conversion` is included as a dependency?

Reply:
Let me double-check (no pun intended)

Reply:
Examples from grepping the source for "double-conversion":

```
qtbase/src/3rdparty/double-conversion/bignum-dtoa.cc:#include <double-conversion/ieee.h>
qtbase/src/corelib/configure.json:            "headers": "double-conversion/double-conversion.h",

```
But
`qtwebengine/src/3rdparty/chromium/third_party/icu/BUILD.gn:    "source/i18n/double-conversion.h",`

From the lines in configure.json I deduce that this here is the correct layout and previously it used the included version of double-conversion

Reply:
Hm, has the Qt5 EC/EB been recently changed? I can't see the configure summary for the Qt5 I installed recently but rebuilding shows it and correctly mentions "Using system DoubleConversion ........ yes" But I'm comparing the log created during build in/tmp and the one installed in the easybuild folder. Maybe the latter doesn't contain the config summary for some reason? Currently rebuilding both QTs using double-conversion and will check what they install and use

Reply:
Ok seems whether Qt configure outputs a summary depends on the moon phase. Run it twice, got one with and one without the summary :/

Reply:
Anyway: I rebuild Qt on top of this and ldd shows that the correct libdouble-conversion is used for both Qt5 ECs that have this dependency

Reply:
And finally: TensorFlow can use this and it does use `#include "double-conversion/double-conversion.h"`